src/index.test.js

```diff
@@ -14,4 +14,8 @@ a,b,c`)).toStrictEqual([['a', 'b', 'c'], ['a', 'b', 'c']]);
 test('find duplicates', () => {
   expect(findDuplicates([['a', 'b', 'b'], ['a', 'b', 'c']])).toStrictEqual(["a"]);
   expect(findDuplicates([['a1', 'b', 'b'], ['a2', 'b', 'c'], ['a1', 'b', 'b'], ['a2', 'b', 'c']])).toStrictEqual(["a1", "a2"]);
```

Comment:
Also your code handles items present more than twice, could be good to show that in the tests, with a test like:

> expect(findDuplicates([['a1', 'b', 'b'], ['a2', 'b', 'c'], ['a1', 'b', 'b'], ['a2', 'b', 'c'], ['a1', 'b', 'b']])).toStrictEqual(["a1", "a2"]);

Reply:
added in the blablablabla line.

Reply:
btw this PR was mostly done as a joke feel free to edit what you want of course :P

Reply:
honestly a very valid PR